src/logo-paths-codegen.js

```diff
@@ -0,0 +1,17 @@
+const revFile = require('rev-file');
```

Comment:
@clxandstuff Overall looks great! I would keep correlation between codegen template and source file naming though 

eg. if the source file name is: `logos.js` let's name codegen file: `logos.codegen.js` instead of `logo-paths-codege` That way it would much easier to reason about the relation between files just by looking at file tree.

Reply:
I like this convention. Let's use it.